storm-client/src/jvm/org/apache/storm/daemon/worker/Worker.java

```diff
@@ -121,6 +121,10 @@ public Worker(Map<String, Object> conf, IContext context, String topologyId, Str
 
         this.topologyConf = ConfigUtils.overrideLoginConfigWithSystemProperty(ConfigUtils.readSupervisorStormConf(conf, topologyId));
 
+        // see STORM-3728.
```

Comment:
Can this comment say something like: "When Pacemaker auth is set to Kerberos, unauthenticated writes are allowed"

Reply:
It is a little more than that.

Currently no matter what value `pacemaker.auth.method` is, write to pacemaker is always allowed. see https://github.com/apache/storm/blob/master/docs/Pacemaker.md#security.

Workers only write to pacemaker, so it can be just "NONE". 

So I figured to use a JIRA ID to explain it.

In the future, we want to 
1) fix worker authentication with pacemaker
2) secure write access on pacemaker

Reply:
Ok. That first line itself (or a short variant) is a good enough comment - "write to pacemaker is currently always allowed, ignores pacemaker.auth.method value"

Reply:
updated. Please check. Thanks